com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Version Updated
```

Comment:
This needs to be removed now.

Reply:
Why do we even add that? Seems like a recipe for merge conflicts :/

Reply:
It's added because we release packages in lockstep, and previously it happened that we released one package and the other package contained no changes at all, which was confusing to users. (we make a quick patch for URP or HDRP but not changes to Core).

 This will be solved with changelog tooling.